openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx

```diff
@@ -611,10 +611,12 @@ const EntityTable = ({
         accessor: 'name',
         width: 300,
         render: (name: Column['name'], record: Column) => (
-          <div className="d-flex break-word">
-            {prepareConstraintIcon(name, record.constraint, tableConstraints)}
-            <span className="m-l-xss">{name}</span>
-          </div>
+          <span className="w-max-90 d-inline-block vertical-align-inherit">
+            <div className="d-flex">
```

Comment:
Can't we use Space component from AntD here?

Reply:
@chirag-madlani  Changes updated as per comments!